blog.md

```diff
@@ -74,4 +114,28 @@ return new FileStreamResult(stream, System.Net.Mime.MediaTypeNames.Application.Z
 {
     FileDownloadName = "remeshed.zip"
 };
-```
\ No newline at end of file
+```
+
+## Testing
+
+So, we have shown how to build and deploy two example nodes for a 3D content pipeline but in order to test this I currently need to carry out the following:
+
+- Take the model and textures from [here](https://github.com/peted70/az-func-blender/tree/master/TestModels) and add all contents of that folder to a zip archive as it is organized in the folder
+- Manually uploaded to blob storage or some other accessible storage
+- Used the resulting uri (I generated a SAS uri using the [Azure portal](https://ms.portal.azure.com/) as input to run the Blender Azure Function running either locally or in Azure and download the resulting zip archive (You could use [Postman](https://www.postman.com/) with a POST/GET request or a browser with a GET request)
+- Upload that zip archive to blob storage manually (so we can get another uri)
+- Use the resulting SAS uri as input to the Simplygon Azure Function running locally on PC or deployed to Azure

```

Comment:
That may look a bit barbarian for many people. Any way to make a real step by step with few captures maybe?

Reply:
Added some step-by-step images